config/ext/rspec.yml

```diff
@@ -896,6 +935,12 @@ RSpec/SubjectStub:
   StyleGuide: https://rspec.rubystyle.guide/#dont-stub-subject
   Reference: https://www.rubydoc.info/gems/rubocop-rspec/RuboCop/Cop/RSpec/SubjectStub
 
+RSpec/UndescriptiveLiteralsDescription:
```

Comment:
https://docs.rubocop.org/rubocop-rspec/cops_rspec.html#rspecundescriptiveliteralsdescription

Sollte imho dem Entwickler überlassen sein :-1:

Reply:
Agree. Please set enabled to false